deepset_cloud_sdk/workflows/sync_client/files.py

```diff
@@ -167,6 +171,45 @@ def upload_texts(
     )
 
 
+def upload_bytes(
+    files: List[DeepsetCloudFileBytes],
+    api_key: Optional[str] = None,
+    api_url: Optional[str] = None,
+    workspace_name: str = DEFAULT_WORKSPACE_NAME,
+    write_mode: WriteMode = WriteMode.KEEP,
+    blocking: bool = True,
+    timeout_s: Optional[int] = None,
+    show_progress: bool = True,
+) -> S3UploadSummary:
+    """Upload texts to deepset Cloud.
```

Comment:
I'd update the doc string here to reflect that all file types are supported. It'd be great to list the supported file types here.